examples/kg-chat/README.md

```diff
@@ -0,0 +1,71 @@
+# PyPi Packages Dependency Chatbot
+
+This example uses a `DependencyGraphAgent` 
+(derived from [`Neo4jChatAgent`](https://github.com/langroid/langroid/blob/main/langroid/agent/special/neo4j/neo4j_chat_agent.py)).
+It auto-generates a `neo4j` knowledge-graph based on the dependency
+structure of a given `PyPi` package. You can then ask the chatbot questions
+about the dependency graph. This agent uses two tools in addition to those 
+already available to `Neo4jChatAgent`:
+
+- GoogleSearchTool to find package version and type information 
+- DepGraphTool to build the dependency graph for a given pkg version, using the API
+   at [DepsDev](https://deps.dev/)
+
+The `Neo4jChatAgent` has access to these tools/function-calls:
+
+- `GraphDatabaseSchema`: get schema of Neo4j knowledge-graph
+- `GenerateCypherQueries`: generate cypher queries to get information from
+   Neo4j knowledge-graph (Cypher is the query language for Neo4j)
+
+
+# Requirements:
+
+This example relies on the `neo4j` Database. The easiest way to get access to neo4j is
+by creating a cloud account at [Neo4j Aura](https://neo4j.com/cloud/platform/aura-graph-database/). OR you
+can use Neo4j Docker image using this command:
+
+```bash
+docker run --rm \
+    --name neo4j \
+    -p 7474:7474 -p 7687:7687 \
+    -e NEO4J_AUTH=neo4j/password \
+    neo4j:latest
+```
+
+Upon creating the account successfully, neo4j will create a text file contains
+account settings, please provide the following information (uri, username,
+password, and database), while creating the constructor `Neo4jChatAgentConfig`. 
+These settings can be set inside the `.env` file as shown in [`.env-template`](../../.env-template)
+
+# Running the example
+
+Run like this:
+```
+python3 examples/kg-chat/dependency_chatbot.py
+```
+
+`DependencyAgent` then will ask you to provide the name of the `PyPi` package.
```

Comment:
this paragraph needs to be updated since we decided to proceed with one agent-2 tools

Reply:
ok please make that change